async-inference/Async-Inference-Walkthrough-SageMaker-Python-SDK.ipynb

```diff
@@ -0,0 +1,652 @@
+{
```

Comment:
<p>Thanks for adding the citation. Please remove the two biostat.mc.vanderbilt.edu links because unfortunately they are broken (and have been for at least months, if not longer)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done